app/models/foreman_openscap/arf_report.rb

```diff
@@ -4,6 +4,7 @@ module ForemanOpenscap
   class ArfReport < ::Report
     include Taxonomix
     include OpenscapProxyExtensions
+    alias_method :destroy_locally, :destroy
```

Comment:
why is this required? do we need to skip some destroy callback?

Reply:
Not a callback, but we have our own [destroy](https://github.com/theforeman/foreman_openscap/blob/master/app/models/foreman_openscap/arf_report.rb#L189) that calls out to proxy to delete the files there. Since the proxy for reports is not set, we just want to destroy what is in db.

Reply:
So why alias is prefered instead of adding a detection of existing object into this destroy? It would skip the deletion if it can't be performed.